components/Code/index.tsx

```diff
@@ -43,28 +43,27 @@ export function Code({ children }: ChildrenProps) {
   return (
     <Tabs items={Object.values(renderedFrameworks)}>
       {Object.keys(renderedFrameworks).map((f) => {
-        const [child] = childs.filter(
-          // @ts-expect-error
-          (c) => (c ? c.type.name === f : false)
-        );
+        // @ts-expect-error - Hacky dynamic child wrangling
+        const child = childs.find((c) => c?.type?.name === f);
 
-        return (
-          child || (
-            <Tabs.Tab key={f}>
-              <p className="font-semibold bg-slate-100 p-6 rounded-lg">
-                {renderedFrameworks[f]} not documented yet. Help us by
-                contributing{" "}
-                <a
-                  className="underline"
-                  target="_blank"
-                  href={`${project.link}/edit/main/docs/pages${router.pathname}.mdx`}
-                >
-                  here
-                </a>
-                .
-              </p>
-            </Tabs.Tab>
-          )
+        // @ts-expect-error - Hacky dynamic child wrangling
+        return !!Object.keys(child?.props ?? {}).length ? (
+          child
+        ) : (
+          <Tabs.Tab key={f}>
+            <p className="p-6 font-semibold rounded-lg bg-slate-100">
+              {renderedFrameworks[f]} not documented yet. Help us by
+              contributing{" "}
+              <a
+                className="underline"
+                target="_blank"
+                href={`${project.link}/edit/main/docs/pages${router.pathname}.mdx`}
+              >
+                here
+              </a>
+              .
+            </p>
+          </Tabs.Tab>
```

Comment:
Yup not the most beautiful thing this child's logic 🙈  

The idea is to not show `Next.js (Pages)` if there's not a code example for it. Tidier with your refactor 💚 🧹